### DISCUSSION TITLE
NER costum model

### DISCUSSION THREAD

**Discussion by Astudnew** [aub-mind/arabert#99] [Q&A]
Hi
There was an example for NER with a custom model. Do delete it or change it? i just find one example with a trainer ?

**Comment by WissamAntoun** [comment]
I removed it, and kept the example here `https://colab.research.google.com/drive/128wX6r_Nz-2LZ_lROeWjWS_yTi96KR7J?usp=sharing` check the section after the Hyper-parameter search.

This is the old one https://colab.research.google.com/drive/1M1yH8rqbxowlSd5R-jr8eBVfGbIyp1zC?usp=sharing

**Reply by Astudnew** [reply]
OK, Thank you.

**Reply by Astudnew** [reply]
please send the sentiment classification example with custom model part
